src/consent-manager-builder/preferences.ts

```diff
@@ -76,4 +76,9 @@ export function savePreferences({
     destinationPreferences,
     customPreferences
   })
+  // NOTE: bc-consent coockie is being used for the same purpose
+  cookies.remove(COOKIE_KEY, {
```

Comment:
I'd like to add remark here. Since we have savePreferences function that is being used I've thought that simply removing tracking-preferences cookie  at the end can be better than deleting/commenting ` cookies.set` method. In this case whole function loses sense. 

What are your thoughts on this?